js/nested-for-loops.js

```diff
@@ -8,7 +8,22 @@ function nestedForLoops (depth, width = depth) {
   let result = "\n";
 
   // Write code here
-
+  
+  for(var y=0;y<depth;y++){
+    for(var x=0;x<width;x++){
+      if(x === width -1){
```

Comment:
So this if conditional can be simplified down to two cases here: `{x: x, y: y}, ` and `{x: x, y: y}` with the only difference being whether it ends with a `, ` or nothing at the end.

Reply:
the second and third have one difference in which '{x:' has a space before it, such as: " {x" versus no space: "{x". Should i still simplify it?

Reply:
Yea I think you can still simplify it by putting the space on the other side of the string so that it really just boils down to two case, one where it ends with the comma and space and one where it ends with nothing after the string.